### PR TITLE
Added functions for using authenticators.

### PR DESCRIPTION
**Pull request by AntonCroner**
Added functions for base32.

### CODE REVIEW

File: authenticator.spec.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { cryptly } from "./index"
+describe("authenticator", () => {
+	it("HS256 authenticator", async () => {
+		const key = "12345678901234567890"
+		const ziggurat = cryptly.authenticator.toQrCode(key, "utily", "testson")
+		console.log("ziggurat: ", ziggurat)
```
Comment (marcushanses) [inline authenticator.spec.ts]: Remove console.log.

File: authenticator.spec.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+		const time = 1706101897967
+		const previous = await cryptly.authenticator.generate(key, time - 30000)
+		const current = await cryptly.authenticator.generate(key, time)
+		expect(previous).toEqual("727592")
+		expect(current).toEqual("097958")
```
Comment (marcushanses) [inline authenticator.spec.ts]: remove consts.

**Review by marcushanses — CHANGES_REQUESTED** [review]

File: authenticator.spec.ts (hunk 1 above)
Comment (AntonCroner) [reply authenticator.spec.ts]: \+

File: authenticator.spec.ts (hunk 2 above)
Comment (AntonCroner) [reply authenticator.spec.ts]: \+

**Review by marcushanses — DISMISSED** [review]

**Comment by AntonCroner** [comment]
Check.

**Review by Aftonsaga — APPROVED** [review]

**Pull request merged**